File: expeditionsjournal-web/src/app/recording.ts
export class Recording {

    /**
     * The GUID of this recording
     */
    id?: string;

    /**
     * The related goals as GUID (in most cases it is one goal)
     */
    goalIds: string[];

    /**
     * The date of the recording (can be different from the item createdDate)
     */
    date: Date;

    /**
     * The amount of the recording
     */
    amount: number;

    /**
     * Recording comment
     */
    comment?: string;

    /**
     * The date this item has been created
     */
    itemCreatedDate?: Date;

}

File: monatsziele-api/Monatsziele.Api/Models/Recording.cs
using System;

namespace Monatsziele.Api.Models
{
    public class Recording
    {

        public Guid Id { get; set; }

        public Guid GoalId { get; set; }

        public DateTime Date { get; set; }

        public double Amount { get; set; }

        public string Comment { get; set; }

        public DateTime ItemCreatedDate { get; set; }

    }
}
